.nvmrc

```diff
@@ -0,0 +1 @@
+12
```

Comment:
Missing newline.
Also, we need at least 12.17, can we somehow encode that? Or perhaps default to 14?

Reply:
I just tested it, `nvm` doesn't support syntax such as `>=12.7` and I don't think its best use is minors.
It would install the latest 12 if you didn't have one installed and would default to the latest 12 installed on your system.
But it makes sense to default to `14` since it's the active LTS.

Reply:
Fixed and commited